pkgs/tools/text/papeer/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "papeer";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "lapwat";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-N0ZOVP64+2iyR04GSgwIGQZ5VSgpwGrsN7+quyob9iA=";
+  };
+
+  vendorHash = "sha256-M9fe0hHC/u4b36V7Ux/Z/hBgZjYS0Ve50ftPLT4CiAg=";
+
+  doCheck = false; # uses network
+
+  meta = {
+    description = "Convert websites into ebooks and markdown";
+    homepage = "https://papeer.tech/";
```

```suggestion
    homepage = "https://papeer.tech/";
    license = lib.licenses.gpl3Plus;
```


https://github.com/lapwat/papeer/issues/8